addons/external_intercom/XEH_preInit.sqf

```diff
@@ -0,0 +1,61 @@
+#include "script_component.hpp"
+
+#include "XEH_PREP.sqf"
+
+[
+    "TFAR_setting_externalIntercomWirelessHeadgear",
+    "EDITBOX",
+    [LSTRING(SETTING_WIRELESS_HEADGEAR_HEADER), LSTRING(SETTING_WIRELESS_HEADGEAR_DESC)],
+    [localize ELSTRING(settings,global), localize LSTRING(SETTINGS_CATEGORY_EXTERNAL_INTERCOM)],
+    ["", false, FUNC(settingForceAlphaNumericArray)],
+    1,
+    {
+        if (isServer) then {
+            TFAR_externalIntercomWirelessHeadgear = ((configFile >> "TFAR_External_Intercom_Wireless_Headgear") call BIS_fnc_getCfgSubClasses) + ([_this] call FUNC(parseClassnamesInput));
```

Comment:
just directly https://community.bistudio.com/wiki/configProperties with "true" condition might be cheaper than calling the BIS func